timeseries/src/autogluon/timeseries/learner.py

```diff
@@ -61,7 +61,7 @@ def fit(
         hyperparameters: Union[str, Dict],
         val_data: Optional[TimeSeriesDataFrame] = None,
         hyperparameter_tune_kwargs: Optional[Union[str, dict]] = None,
-        time_limit: Optional[int] = None,
+        time_limit: Optional[float] = None,
```

Comment:
After the initial review of all "breaking" changes, let's also change the type hints to `dict`, `list`, etc. This will make the docs look better https://auto.gluon.ai/stable/api/autogluon.timeseries.TimeSeriesPredictor.html

Reply:
I'm planning to do that library-wide in a future PR if that's OK